internal/tools/networkagent/main.go

```diff
@@ -26,8 +26,9 @@ import (
 var (
 	configPath = flag.String("config", "", "Path to JSON configuration file.")
 
-	prefixLength = "24"
-	ipVersion    = "4"
+	prefixLengthAsInt uint32 = 24
+	prefixLength             = "24"
+	ipVersion                = "4"
```

Comment:
should these be `const`?

Reply:
They can be. configpath needs to be a var but the others can be const.